.eslintrc.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
     'no-restricted-syntax': 'off',
     'no-plusplus': 'off',
     'func-names': 'off',
+    'import/no-cycle': 'off',
```

Comment:
remove this eslint rule.

Reply:
Ok. I didn't remember if it was added by me. I'll remove this.